_sass/components/_eiti-slider.scss

```diff
@@ -34,11 +34,11 @@ eiti-slider {
     z-index: 100;
 
     &:after {
-      border-color:  #ccc;
+      border-color: #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.